.github/workflows/desktop-build.yml

```diff
@@ -273,15 +273,13 @@ jobs:
       matrix:
         include:
           - target: 7
-            qt_version: 5.15.*
+            qt_version: 5.15.14
```

Comment:
If this change is reverted, we are good to merge IMHO. 👍 

We have a new issue to track the SSL incompatibility rooted in unsupported Qt5.

Reply:
ok